grakn-dist/src/grakn

```diff
@@ -68,6 +70,15 @@ update_classpath_global_var() {
   CLASSPATH="$CLASSPATH":"${GRAKN_HOME}"/services/grakn
 }
 
+ps_ef_grep_processname() {
+  ps -ef | grep -v grep | awk '{print $2" "$(NF-1)}' | grep "$1"
+}
+
+
+ps_ef_grep_processname_by_java_classname() {
+  ps -ef | grep -v grep | awk '{print $2" "$(NF)}' | grep "$1"
```

Comment:
`jps` should give you what you want

Reply:
@burukuru as far as I know, `jps` isn't part of OpenJDK which we claim to support

Reply:
oh that sucks

Reply:
yup. the joy of maintaining platform compatibility...